libs/LedKit/tests/LedKit_test.cpp

```diff
@@ -4,8 +4,6 @@
 
 #include "LedKit.h"
 
-#include "rtos/tests/UNITTESTS/doubles/Thread_stub.h"
```

Comment:
the emoji :coffin: is really for dead code. This is not dead code.

it's more of cleanup :art:

Reply:
or refactor ♻️